libs/core/coroutines/include/hpx/coroutines/detail/context_generic_context.hpp

```diff
@@ -90,7 +90,9 @@ namespace hpx { namespace threads { namespace coroutines {
 
             void* allocate(std::size_t size) const
             {
-#if defined(_POSIX_VERSION)
+       // Condition excludes MacOS/M1 from using posix mmap
+#if defined(_POSIX_VERSION) && 
+    !(defined(__ARM64_ARCH_8__) && defined(__APPLE__))
```

Comment:
This will not compile as it misses the `\` at the end of the line. Just use clang-format itself on that file to avoid this kind of problems.

Reply:
ahh ok i see.. im sorry. I use clang format but I think my IDE had different setting, but I turned it off.

Reply:
adjusted w/ clang-format